### PR TITLE
155 when a user is blocked their black cards dont show up for the other user anymore

### PR DESCRIPTION
**Pull request by anhuwi**
1) bugfix: past games get deleted when nobody played on them
2) blocked users can not play each other anymore

### CODE REVIEW

File: src/main/java/ch/uzh/ifi/hase/soprafs22/service/UserService.java [hunk 1]
```diff
@@ -40,7 +40,7 @@ public class UserService {
   private final GameService gameService;
   private boolean areInstantiatedDemoUsers = false;
   private static final String UNIQUE_VIOLATION = "Uniqueness Violation Occurred";
-  private static final Long GAME_DURATION = Time.ONE_DAY;
+  private static final Long GAME_DURATION = 2*Time.ONE_MINUTE;
```
Comment (serainaschraff) [inline src/main/java/ch/uzh/ifi/hase/soprafs22/service/UserService.java:43]: shouldn't this be ONE_DAY again?

File: src/main/java/ch/uzh/ifi/hase/soprafs22/service/UserService.java [hunk 2]
```diff
@@ -882,4 +882,25 @@ public void blockUser(long userId, long otherUserId) {
     userRepository.saveAndFlush(user);
     userRepository.saveAndFlush(otherUser);
   }
+
+    /**
+     * recursive function to delete all past games without plays on it
+     * @param user user where games have to be deleted
+     */
+    public void deleteAllEmptyPastGames(User user) {
```
Comment (daveking7) [inline src/main/java/ch/uzh/ifi/hase/soprafs22/service/UserService.java:890]: method name does not correspond with what it does: recursion stops as soon as it encounters a game with plays, but there can be newer games with no plays that don't get deleted.

**Review by daveking7 — COMMENTED** [review]
Looks fine, two small things

File: src/main/java/ch/uzh/ifi/hase/soprafs22/service/UserService.java (hunk 1 above)
Comment (anhuwi) [reply src/main/java/ch/uzh/ifi/hase/soprafs22/service/UserService.java:43]: sooory!

**Pull request merged**